addons/miscFixes/patchPLA/config.cpp

```diff
@@ -0,0 +1,593 @@
+                radius = 0.33;
+                armor = -250;
+                minimalHit = -0.016;
+                explosionShielding = 4;
+                passThrough = 0;
+            };
+            class HitRF2Wheel: HitRF2Wheel {
+                radius = 0.33;
+                armor = -250;
+                minimalHit = -0.016;
+                explosionShielding = 4;
+                passThrough = 0;
+            };
+            class HitRMWheel: HitRMWheel {
+                radius = 0.33;
+                armor = -250;
+                minimalHit = -0.016;
+                explosionShielding = 4;
+                passThrough = 0;
+            };
+            class HitRBWheel: HitRBWheel {
+                radius = 0.33;
+                armor = -250;
+                minimalHit = -0.016;
+                explosionShielding = 4;
+                passThrough = 0;
+            };
+            class HitFuel: HitFuel {
+                armor = 0.75;
+                passThrough = 0.5;
+                minimalHit = 0.1;
+                explosionShielding = 0.6;
+                radius = 0.16;
+            };
+            class HitArmor {
+                armor = 0.5; //formerly 10
+                passThrough = 0.5; //formerly 0 (0.5 causes tires to transfer dmg to hull/turret?)
+                minimalHit = 0.1;
+                explosionShielding = 2;
+                radius = 0.1;
+            };
+        };
+        class Turrets: Turrets {
+            class MainTurret: MainTurret {
+                class Turrets: Turrets {
+                    class CommanderOptics: CommanderOptics {
+                        class HitPoints {
+                            class HitTurret {
+                                armor = 0.3;
+                                passThrough = 0;
+                                minimalHit = 0.03;
+                                explosionShielding = 0.6;
+                                radius = 0.25;
+                            };
+                        };
+                    };
+                };
+                class HitPoints {
+                    class HitTurret {
+                        armor = 1.5; //formerly 0.8
+                        passThrough = 0; //formerly 0
+                        minimalHit = 0.1;
+                        explosionShielding = 0.2;
+                        radius = 0.3;
+                        isTurret = 1;
+                    };
+                    class HitGun {
+                        armor = 1.5;
+                        passThrough = 0; //formerly 0
+                        minimalHit = 0.1;
+                        explosionShielding = 0.4;
+                        radius = 0.25;
+                        isGun = 1;
+                    };
+                };
+            };
+        };
+    };
+    class O_ZTL11: ZTL11_base {};
+    class ZBD04A_base: Tank_F {
+        damageResistance = 0.02;
+        crewVulnerable = 1;
+        transportSoldier = 7;
```

Comment:
what is transport soldier?

Reply:
https://community.bistudio.com/wiki/CfgVehicles_Config_Reference#transportSoldier

Reply:
unintentional - removing now!